### PR TITLE
Mapping between ruby base and ruby text in complex cases (#247).

### LINKED ISSUE #247: How to handle mismatched sequences in complex ruby
I may have missed it, but i think the spec doesn't say anything about what should happen if, for example, an rbc contains 2 rb items and the rtc contains 4 rt items, or vice versa.

### PR DESCRIPTION
**Pull request by cconcolato**
closes #247 by indicating that mapping btw rb and rt in complex cases is implementation dependent

### CODE REVIEW

File: spec/ttml2.xml [hunk 1]
```diff
@@ -11650,6 +11650,9 @@ base delimiter text delimiter =
 &lt;/span&gt;
 </eg>
 </note>
+<note role="clarification">
+<p>The syntax of TTML allows using multiple Ruby Base Content within a Ruby Base Container and multiple Ruby Text Content within a Ruby Text Container all within the same Ruby Container. This version of TTML leaves the mapping between Ruby Base Content and Ruby Text Content implementation defined.</p>
```
Comment (nigelmegitt) [inline spec/ttml2.xml]: Two things:

1. (a nit) The line is very long - it's easier to review if split.
2. The second sentence needs to be scoped to situations where there is not a 1:1 match - this is not apparent from the context set by the first sentence.

Comment (cconcolato) [reply spec/ttml2.xml]: Will do.

Comment (cconcolato) [reply spec/ttml2.xml]: @nigelmegitt Done. Can you check?

File: spec/ttml2.xml [hunk 2]
```diff
@@ -11317,7 +11317,10 @@ base delimiter text delimiter =
 </eg>
 </note>
 <note role="clarification">
-<p>The syntax of TTML allows using multiple Ruby Base Content within a Ruby Base Container and multiple Ruby Text Content within a Ruby Text Container all within the same Ruby Container. This version of TTML leaves the mapping between Ruby Base Content and Ruby Text Content implementation defined.</p> 
+<p>The syntax of TTML allows using multiple Ruby Base Content within a Ruby Base Container and multiple Ruby Text Content within 
+a Ruby Text Container all within the same Ruby Container. This version of TTML leaves the mapping between Ruby Base Content and 
+Ruby Text Content implementation defined when the number of Ruby Text Content does not match the number of Ruby Base Content in
```
Comment (nigelmegitt) [inline spec/ttml2.xml]: s/defined/undefined

Comment (nigelmegitt) [reply spec/ttml2.xml]: I would also edit "match" to "equal" for clarity

Comment (cconcolato) [reply spec/ttml2.xml]: @nigelmegitt "implementation defined" is already used once in the spec. "implementation dependent" is used 10 times. "implementation specific" is used 8 times. But "implementation undefined" is not used and I don't think it makes sense to say "implementation undefined".

Comment (nigelmegitt) [reply spec/ttml2.xml]: @cconcolato sorry, you're right, I meant change "implementation defined" to "undefined".

**Pull request merged**